tests/unit_test/action/test_store_page_action.py

```
@@ -0,0 +1,191 @@
+
+    def test_story_step_type_enum_has_store_page_action(self):
+        assert StoryStepType.store_page_action.value == "STORE_PAGE_ACTION"
+
+    def test_action_factory_has_store_page_action(self):
+        from kairon.actions.definitions.factory import ActionFactory
+        assert ActionType.store_page_action.value in ActionFactory._ActionFactory__implementations
+
+    # ─── model validation ─────────────────────────────────────────────────────
+
+    def test_store_page_action_validate_empty_name(self):
+        with pytest.raises(Exception):
```

Ruff (0.16.0)

[warning] Do not assert blind exception: `Exception`

(B017)